index.html

```diff
@@ -0,0 +1,153 @@
+<!-- Header & Navigation -->
+		<header class="colorHeaderFooter fontCursive">
+			<nav class="fontSizeNav ">
+				<a href="#About">About</a>
+				<a href="#Portfolio">Portfolio</a>
+				<a href="#Skills">Skills</a>
+				<a href="#Contact">Contact</a>
+			</nav>
+		</header>
+
+		<main>
+<!-- Section Name & Background Picture -->			
+			<section class="colorBackgroundDark bkgImageCoding">
+				<!-- <h1 class="fontCursive colorFontSections">About</h1> -->
+				<!-- bkg img not working as expecte -->
+				<div class="containerFlex">
+					<article class="articleBox contentIntro">
+						<div>
+							<p class="textAboutMe fontCursive fontSizeMyName">Davin Hayden</p>
+						</div>
+						<div>
+							<p class="textAboutMe">Web Developer in UK & NZ</p>
+						</div>	
+					</article>
+					
+				</div>
+			</section>
+			
+<!-- Section About Me -->
+			<section class="colorBackgroundDark">
+				<a name="About">
+				<h1 class="fontCursive colorFontSections">About</h1>
+				</a>
+				<!-- bkg img not working as expecte -->
+				<div class="containerFlex ">
+					<article class="articleBox">
+						<p>I'm an IT Professional with over 12 years industry experience in New Zealand and the UK, pivoting into web development.</p>
+						
+					</article>
+<!--check for redundant class? below -->
+					<article class="articleBox"> 
+						<p>Interests:</p>
+						<p>making short films, psychology, travelling, science fiction</p>
+					</article>
+					<article class="articleBox">
+						<p>Education:</p>
+						<p>BSc Hons Computer Science integrated with IBM</p>
+						<p>Certificate in Applied Film & TV Production</p>
+					</article>
+				</div>
+			</section>
+			
+<!-- Section Portfolio-->
+			<section class="colorBackgroundLight">
+				<a name="Portfolio">
+				<h1 class="fontCursive colorFontSections">Portfolio</h1>
+				</a>
+				<div class="containerFlex">
+					<article class="articleBox">
+						<h2>Project A</h2>
+						<p>"Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. </p>
+					</article>
+					<article class="articleBox">
+						<h2>Project B</h2>
+						<p>"Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. </p>
+					</article>
+					<article class="articleBox">
+						<h2>Project C</h2>
+						<p>"Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. </p>
+					</article>
+				</div>
+			</section>
+			
+<!-- Section Skills-->
+			<section class="colorBackgroundDark">
+				<a name="Skills">
+				<h1 class="fontCursive colorFontSections">Skills</h1>
```

Comment:
same as above

Reply:
As above, I've chosen to use h1 for top level headings which will get repeated eg About, Portfolio etc